### PR TITLE
Bugfix - User Modules

### PR DESCRIPTION
**Pull request by raja-2604**
**Title**
Bugfix - User Modules

**Description**

- I have updated the edit user modules.
- Dropdown enabled the edit option in edit user module.
- Enabled as user can't edit and delete user data.

**Reviewers**
@chandran-kannan @harish-flyers @JayshreeFlyers @Roslind-Vasantha @andruroshan @yuvaJax24

### CODE REVIEW

File: src/services/userServices.ts [hunk 1]
```diff
@@ -16,15 +16,15 @@ export const USER_SERVICES = {
   },
   getAllUsers: async (page: number, limit: number) => {
     try {
-      const res = await apiInstance.get(`${SERVICES.user.get}?page=${page || 1}&limit=${limit || 10}`);
+      const res = await apiInstance.get(`${SERVICES.user.get}?page=${page || 1}&limit=${limit || 1000}`);
       return res.data;
```
Comment (harish-flyers) [inline src/services/userServices.ts:20]: Please have a proper limit @raja-2604

File: src/pages/users/UsersList.tsx [hunk 2]
```diff
@@ -133,14 +156,54 @@ function UsersList() {
...
+          {
+            plantId: (selectedUser.plants.length && selectedUser.plants[0]?.plantId) || '',
+          },
+        ],
+      },
     };
+    if (selectedUser.password) {
+      body.password = selectedUser.password;
+    }
+    console.log(body, 'user update');
```
Comment (harish-flyers) [inline src/pages/users/UsersList.tsx:197]: Remove consoles @raja-2604

**Review by harish-flyers — COMMENTED** [review]
Pls check comments

File: src/pages/users/UsersList.tsx (hunk 2 above)
Comment (raja-2604) [reply src/pages/users/UsersList.tsx:197]: > Remove consoles @raja-2604

removed @harish-flyers bro

**Comment by raja-2604** [comment]
> Pls check comments

completed all comments @harish-flyers bro

File: src/services/userServices.ts (hunk 1 above)
Comment (raja-2604) [reply src/services/userServices.ts:20]: > Please have a proper limit @raja-2604

@harish-flyers as chandran bro mentioned to increase the page limit due to not completed the pagination from the backend and will revert once the updated.

**Review by harish-flyers — APPROVED** [review]
Approved

**Comment by chandran-kannan** [comment]
Thanks for the review @harish-flyers

**Pull request merged**